e2e/run_e2e_stage.sh

```diff
@@ -9,6 +9,8 @@
 # By default AWS provider on `us-east-1` region is used because stage Data Plane is configured that way.
 #
 
+echo "ACS fleet manager base url: $ACS_FLEET_MANAGER_ENDPOINT"
```

Comment:
curly brackets for consistency with the rest of the file?

```suggestion
echo "ACS fleet manager base url: ${ACS_FLEET_MANAGER_ENDPOINT}"
```